src/examples/hotgym/HelloSPTP.cpp

```diff
@@ -80,6 +73,7 @@ Real64 BenchmarkHotgym::run(UInt EPOCHS, bool useSPlocal, bool useSPglobal, bool
   encParams.maximum = 100.0;
   encParams.size = DIM_INPUT;
   ScalarEncoder enc( encParams );
+  Random rnd(1); //FIXME fix bug in Random, where static_gen must not be used! change this '1' and all breaks
```

Comment:
Can you remove this comment since IIRC this bug was closed as not a bug?  Maybe replace the comment with a note explaining why this test needs a hardcoded seed:
`// Use this seed to check deterministic output.`